test/python/result/test_mitigators.py

```diff
@@ -442,6 +442,16 @@ def test_expectation_value_endian(self):
             expval, _ = mitigator.expectation_value(counts, diagonal="IZ", qubits=[0, 1])
             self.assertAlmostEqual(expval, -1.0, places=0)
 
+    def test_quasi_probabilities_shots_passing(self):
```

Comment:
Thanks for adding a test. I think that it should also contain a docstring to explain what the test is doing.
Could you add a comment explaining how you start with counts that have 1024 shots, and then change it to 1025 shots?

Reply:
Looks good! there is only one minor lint error to fix now (the comment line is too long).